src/Engines/nativeEngine.ts

```diff
@@ -301,6 +303,22 @@ export class NativeEngine extends Engine {
         return null;
     }
 
+    /**
+     * Gets host window
+     * @returns the host window object
+     */
+    public getHostWindow(): Nullable<Window> {
+        return window;
+    }
+
+    public updateDynamicTexture(texture: Nullable<InternalTexture>, canvas: HTMLCanvasElement, invertY: boolean, premulAlpha: boolean = false, format?: number): void {
+        // TODO
+    }

```

Comment:
Not sure about this one. It has the HTML canvas which will not be available in native.

Reply:
Maybe we should use TexImageSource here and only support ImageData